src/www/vpn_openvpn_server.php

```diff
@@ -1077,7 +1077,7 @@
                     </tr>
                     <tr>
                       <td style="width:22%" id="ipv4_tunnel_network"><a id="help_for_tunnel_network" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("IPv4 Tunnel Network"); ?></td>
-                      <td witdh="78%">
+                      <td width="78%">
```

Comment:
fixed the remaining ones via https://github.com/opnsense/core/commit/1bfc8ef5df

Reply:
maybe this and the other one should be changed to style= as well?

$ git grep td.witdh
src/www/vpn_openvpn_server.php:                      <td witdh="78%">
$ git grep td.width
src/www/interfaces.php:                          <td width="22%"><i class="fa fa-info-circle text-muted"></i> <?=gettext('Service Provider') ?></td>

Reply:
Done.